package.json

```diff
@@ -4,93 +4,67 @@
   "dependencies": {
-    "@oclif/core": "^2.15.0",
+    "@oclif/core": "^3.7.1",
     "@salesforce/core": "^5.3.1",
     "@salesforce/kit": "^3.0.14",
-    "@salesforce/sf-plugins-core": "^3.1.28",
-    "got": "^11.8.6",
+    "@salesforce/sf-plugins-core": "^4",
+    "got": "^13.0.0",
     "marked": "^4.3.0",
     "marked-terminal": "^4.2.0",
     "open": "^8.4.2",
     "proxy-agent": "^6.3.1",
-    "semver": "^7.5.4",
-    "tslib": "^2"
+    "semver": "^7.5.4"
   },
   "devDependencies": {
-    "@oclif/plugin-command-snapshot": "^4.0.14",
-    "@salesforce/cli-plugins-testkit": "^4.4.10",
-    "@salesforce/dev-config": "^4.0.1",
-    "@salesforce/dev-scripts": "^5.12.2",
-    "@salesforce/plugin-command-reference": "^3.0.45",
-    "@salesforce/prettier-config": "^0.0.3",
-    "@salesforce/ts-sinon": "^1.4.19",
-    "@swc/core": "1.3.39",
+    "@oclif/plugin-command-snapshot": "^5",
+    "@salesforce/cli-plugins-testkit": "^5.0.0-dev.1",
+    "@salesforce/dev-scripts": "^6.0.1",
+    "@salesforce/plugin-command-reference": "^3.0.33",
+    "@salesforce/ts-sinon": "^1.4.18",
     "@types/fs-extra": "^9.0.13",
     "@types/marked": "^4.0.8",
     "@types/marked-terminal": "^3.1.3",
     "@types/semver": "^7.5.4",
     "@types/sinon-chai": "^3.2.11",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.62.0",
-    "chai": "^4.3.10",
-    "eslint": "^8.52.0",
-    "eslint-config-prettier": "^8.10.0",
-    "eslint-config-salesforce": "^2.0.2",
-    "eslint-config-salesforce-license": "^0.2.0",
-    "eslint-config-salesforce-typescript": "^2.0.0",
-    "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-jsdoc": "^46.8.2",
     "eslint-plugin-sf-plugin": "^1.16.10",
-    "husky": "^7.0.4",
-    "mocha": "^9.1.3",
-    "nyc": "^15.1.0",
-    "oclif": "^3.17.2",
-    "prettier": "^2.8.8",
-    "pretty-quick": "^3.1.0",
+    "oclif": "^4.0.3",
     "shx": "0.3.4",
-    "sinon": "^11.1.1",
     "sinon-chai": "^3.7.0",
-    "ts-node": "^10.4.0",
-    "typescript": "^5.2.2",
-    "wireit": "^0.14.1"
+    "typescript": "^5.2.2"
   },
   "config": {},
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
   },
   "files": [
     "/lib",
     "/messages",
-    "/oclif.manifest.json"
+    "/oclif.manifest.json",
+    "/oclif.lock"
   ],
   "homepage": "https://github.com/salesforcecli/plugin-info",
   "keywords": [
     "force",
     "salesforce",
-    "sfdx",
     "salesforcedx",
-    "sfdx-plugin"
+    "sf-plugin",
+    "sf",
+    "sfdx-plugin",
+    "sfdx"
   ],
   "license": "BSD-3-Clause",
   "oclif": {
     "commands": "./lib/commands",
     "additionalHelpFlags": [
       "-h"
     ],
-    "bin": "sfdx",
+    "bin": "sf",
     "devPlugins": [
       "@oclif/plugin-help",
       "@oclif/plugin-command-snapshot",
       "@salesforce/plugin-command-reference"
     ],
-    "hooks": {
-      "init": [
-        "./lib/hooks/init/init_doctor.js"
-      ]
-    },
```

Comment:
Why is this hook removed?

Reply:
It's not needed because we can initialize the `Doctor` class at the beginning of the `doctor` command execution. Also hooks are fairly expensive in terms of performance, especially the `init` hooks since they'll run on every single command execution, even `--help` and `--version`